app/views/layouts/_about_modal.html.haml

```diff
@@ -0,0 +1,38 @@
+.modal.fade#aboutModal{ 'tabindex': '-1', 'role': 'dialog', 'aria-labelledby': 'exampleModalLAbel?', 'aria-hidden': 'true' }
+  .modal-dialog{ 'role': 'document'}
+    .modal-content
+      .modal-header
+        %h5.modal-title.inline
+          Serial Search Inspector
+        %button.close{ 'type': 'close', 'data-dismiss': 'modal', 'aria-label': 'Close' }
+          %span{ 'aria-hidden': true }
+            &times;
+      .modal-body
+        %p
+          The Serial Search Inspector is a new tool that allows Bike Index to monitor
+          serial number searches on the registry. This allows us to see a little more
+          into how people use the Bike Index and provides additional information to you.
+        %p
+          How it works: The Inspector functions in conjunction with Bike Index. When a
+          serial number is searched for on the Bike Index, we create a log of that
+          search and share it with the Inspector. The Inspector then, determined by the
+          validity of the search, checks if that serial matches to a registered bike on
+          the registry. If there is a match, we link that search and bike together.
+          This lets you see when a bike registered to you is searched for.
+        %p
+          There are a few things to keep in mind regarding this information.
+
+        %p
+          First, it is possible, though unlikely, that two or more bikes could share a
```

Comment:
You should add a note about the fact that it might be a partial serial match here.